src/js/views/patients/patient/flow/flow_views.js

```diff
@@ -171,14 +175,23 @@ const ActionItemView = View.extend({
     this.showDueTime();
     this.showForm();
   },
+  toggleSelected(isSelected) {
+    const isBeingEdited = this.state.isBeingEdited(this.model);
+
+    if (isBeingEdited) return;
+
+    this.$el.toggleClass('is-selected', isSelected);
+  },
```

Comment:
`toggleSelected()` adds or removes the `.is-selected` class whenever the checkbox for that action is clicked (or the bulk select select-all or un-select controls are used).

If the action is currently being edited, this function will `return` without doing anything.

For this function to know whether or not the action is being edited, I needed to add a `state.actionBeingEdited` value that holds the `id` of the action being edited. And the `state.isBeingEdited()` function checks to see if that `id` matches the action `id` that was selected/un-selected via the checkbox.